New Template - hp_procurve_show_interfaces

adraf82 opened the pull request:
##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - New Template Pull Request


##### COMPONENT
<!--- Name of the template, os and command  -->
hp_procurve_show_interfaces.textfsm

##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
I have added a new template for the 'show interfaces' command for a hp procurve device.

<!---
If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

<!-- Paste verbatim command output below, e.g. before and after your change -->
```

```

adraf82 commented:
Hi, has anyone been able to review my commits yet? Please can you advise if any amendments need to be made. This is my first pull request on GitHub. :-) Thanks

jmcgill298 commented:
Thanks Adraf82. The tests are failing, but I believe that if you rebase this with the current master, then they will likely pass. Please let us know if you need help with rebasing.

jmcgill298 commented on templates/index:
```diff
@@ -373,6 +373,7 @@ hp_comware_display_vlan_brief.textfsm, .*, hp_comware, di[[splay]] v[[lan]] b[[r
 hp_comware_display_clock.textfsm, .*, hp_comware, di[[splay]] clo[[ck]]
 hp_comware_display_arp.textfsm, .*, hp_comware, di[[splay]] a[[rp]]
 
+hp_procurve_show_interfaces.textfsm, .*, hp_procurve, sh[[ow]] int[[erfaces]]
```
Please follow the entry rules:

```
# Rules of Ordering:
#  - OS in alphbetical order
#  - Command in length other
#  - When Length is the same, use alphabetical order
#  - Keep space between OS's
#
```

adraf82 commented:
Thanks for the reply Jacob. I have corrected the order for my template within the index file and have committed the change for merging.

I have rebased my repo and the test that was previously failing has passed.

Thanks

Adeel

Sent from my iPhone

On 20 Oct 2020, at 23:11, Jacob McGill <notifications@github.com> wrote:

﻿

Thanks Adraf82. The tests are failing, but I believe that if you rebase this with the current master, then they will likely pass. Please let us know if you need help with rebasing.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/networktocode/ntc-templates/pull/796#issuecomment-713169481>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANIVDFYA2ZR3W4ALHRKBW2LSLYDJBANCNFSM4QNNGBOA>.

jmcgill298 commented:
Thanks @adraf82
